src/Storages/MergeTree/KeyCondition.h

```diff
@@ -48,10 +48,26 @@ class KeyCondition
         const ExpressionActionsPtr & key_expr,
         bool single_point_ = false);
 
+    struct BloomFilterData
+    {
+        using HashesForColumns = std::vector<std::vector<uint64_t>>;
```

Comment:
I guess this needs to be size_t to get the darwin build to work. (Annoying that it pretends that there are two different unsigned 64-bit types. Just add static_cast-s until it works.)